spec/lib/data_update_scripts/re_index_users_to_elasticsearch_spec.rb

```diff
@@ -1,11 +1,13 @@
 require "rails_helper"
-require Rails.root.join("lib/data_update_scripts/20200406213152_re_index_users_to_elasticsearch.rb")
+require Rails.root.join("lib/data_update_scripts/20200519142908_re_index_feed_content_and_users_to_elasticsearch.rb")
 
-describe DataUpdateScripts::ReIndexUsersToElasticsearch, elasticsearch: "User" do
+describe DataUpdateScripts::ReIndexFeedContentAndUsersToElasticsearch, elasticsearch: "User" do
```

Comment:
Describing a different class than the actual data update script is technically not kosher according to Rubocop, but I don't have strong opinions here 🤷‍♂️ It was either do this and ignore Rubocop's offenses, or make a new data update script file.